src/easylink/implementation.py

```diff
@@ -20,7 +20,7 @@ def __init__(
         self.name = config.get_implementation_name(step.name)
         self.environment_variables = config.pipeline["steps"][self.step.name][
             "implementation"
-        ].get("configuration", {})
+        ]["configuration"].to_dict()
```

Comment:
Is it possible for the key `"configuration"` not to exist?

Reply:
The config constructor now sets this even if not specified by the user:
`            self.update(
                {"pipeline": {"steps": {step: {"implementation": {"configuration": {}}}}}},
                layer="default",
            )`